app/src/ui/lib/bytes.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Manipulate bytes for displaying purposes.
+ */
+export class Bytes {
```

Comment:
I don't think we need a class for this. Can we turn these into free-standing functions?

Reply:
@iAmWillShepherd ok, I've refactored this